### PR TITLE
i18n for kubevirt basic VM wizard

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/create-vm/forms/boot-source-form-reducer.ts [hunk 1]
```diff
@@ -107,7 +106,8 @@ export const bootFormReducer = (
   };
   let isValid: boolean;
   const sizeValidation = validatePositiveInteger(`${newState.size?.value.value}`, {
-    subject: `${PersistentVolumeClaimModel.label} size`,
+    // t('kubevirt-plugin~Persistent Volume Claim size')
+    subject: `kubevirt-plugin~Persistent Volume Claim size`,
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/create-vm/forms/boot-source-form-reducer.ts:110]: does it make sense here to replace `subject` with `subjectKey` to make sure users know it need translation when using it ?

File: frontend/packages/kubevirt-plugin/src/components/create-vm/forms/create-vm-form.tsx [hunk 2]
```diff
@@ -154,8 +156,8 @@ export const CreateVMForm: React.FC<CreateVMFormProps> = ({
       }
       return aCPU - bCPU;
     })
-    .reduce((acc, t) => {
-      const flavor = getTemplateFlavorDesc(t);
+    .reduce((acc, tmp) => {
+      const flavor = getTemplateFlavorDesc(tmp);
       acc[flavor] = t;
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/create-vm/forms/create-vm-form.tsx:161]:
```suggestion
      acc[flavor] = tmp;
```

File: frontend/packages/kubevirt-plugin/src/components/create-vm/forms/create-vm-form.tsx [hunk 3]
```diff
@@ -194,19 +196,21 @@ export const CreateVMForm: React.FC<CreateVMFormProps> = ({
...
       <StackItem>
-        You are creating a virtual machine from the <b>{getTemplateName(template)}</b> template.
+        <Trans t={t} ns="kubevirt-plugin">
+          You are creating a virtual machine from the <b>{getTemplateName(template)}</b> template.
+        </Trans>
       </StackItem>
       <StackItem>
         <Form onSubmit={preventDefault}>
-          <FormRow fieldId="vm-namespace" title={useProjects ? 'Project' : 'Namespace'} isRequired>
+          <FormRow fieldId="vm-namespace" title={t('kubevirt-plugin~Project')} isRequired>
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/create-vm/forms/create-vm-form.tsx:205]:
```suggestion
          <FormRow fieldId="vm-namespace" title={useProjects ?  t('kubevirt-plugin~Project') : t('kubevirt-plugin~Namespace')} isRequired>
```

File: frontend/packages/kubevirt-plugin/src/components/create-vm/forms/create-vm-form.tsx (hunk 2 above)
Comment (rawagner) [reply frontend/packages/kubevirt-plugin/src/components/create-vm/forms/create-vm-form.tsx:161]: good catch! fixed

File: frontend/packages/kubevirt-plugin/src/components/create-vm/forms/boot-source-form-reducer.ts (hunk 1 above)
Comment (rawagner) [reply frontend/packages/kubevirt-plugin/src/components/create-vm/forms/boot-source-form-reducer.ts:110]: actually the whole validate* logic is constructing sentences which is not good. We will have to rework it. I've removed the translation for now and we will need to address it it separate PR.

File: frontend/packages/kubevirt-plugin/src/components/create-vm/forms/create-vm-form.tsx (hunk 3 above)
Comment (rawagner) [reply frontend/packages/kubevirt-plugin/src/components/create-vm/forms/create-vm-form.tsx:205]: fixed

**Pull request merged**